xwalk.gyp

```diff
@@ -363,6 +365,11 @@
             '../build/linux/system.gyp:fontconfig',
             '../build/linux/system.gyp:dbus',
           ],
+          'link_settings': {
```

Comment:
Please add this to `build/system.gyp` instead so that that the "crosswalk-libs" Tizen RPM package does not end up having to depend on libnotify.